governance.qmd

```diff
@@ -0,0 +1,96 @@
+---
+title: "R-Multiverse Governance"
+---
+
+The R-Multiverse project is run by volunteers in a collaborative and open way. Portions of this governance policy have been inspired by that of the Apache Software Foundation.
```

Comment:
I'd really encourage adding a new line for each sentence, for nicer Git diff. :grin: